src/openvic-simulation/country/CountryInstance.cpp

```diff
@@ -56,6 +58,11 @@ CountryInstance::CountryInstance(
 
 	/* Budget */
 	cash_stockpile { 0 },
+	poor_tax_rate_slider {50},
```

Comment:
```suggestion
	poor_tax_rate_slider { 50 },
```

Please make the formatting here consistent, each entry in the member initialiser list should have spaces between the variable name and the `{`, and between the starting value and the `{}`